server/enterprise/message_export/shared/shared.go

```diff
@@ -59,13 +65,21 @@ type BackendParams struct {
 	HtmlTemplates         *templates.Container
 }
 
+type WriteExportResult struct {
+	TransferringFilesMs int64
+	ProcessingXmlMs     int64
+	TransferringZipMs   int64
```

Comment:
0/5 the `Ms` suffix is a bit hard to understand. Maybe add a comment and remove the `Ms` part?

Reply:
I think that was a comment from an earlier PR, so I think I'll keep them for clarity about the units.